src/shapes/Circle.ts

```diff
@@ -169,12 +181,13 @@ export class Circle<
         startY = sin(start) * radius,
         endX = cos(end) * radius,
         endY = sin(end) * radius,
-        largeFlag = angle > 180 ? '1' : '0';
+        largeFlag = angle > 180 ? '1' : '0',
+        sweepFlag = this.counterclockwise ? '0' : '1';
       return [
         `<path d="M ${startX} ${startY}`,
         ` A ${radius} ${radius}`,
         ' 0 ',
-        `${largeFlag} 1`,
+        `${largeFlag} ${sweepFlag}`,
         ` ${endX} ${endY}`,
         '" ',
```

Comment:
since you are here, it shoud be possible to make a unique string of this. Just common parts needs to be broken away in its own array element. 
```js
`<path d="M ${startX} ${startY} A ${radius} ${radius} 0 ${largeFlag} ${sweepFlag} ${endX} ${endY}" ` 
```